x-pack/test/functional/apps/painless_lab/painless_lab_flyout.ts

```diff
@@ -0,0 +1,34 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import expect from '@kbn/expect';
+import { FtrProviderContext } from '../../ftr_provider_context';
+
+export default function ({ getService, getPageObjects }: FtrProviderContext) {
+  const retry = getService('retry');
+  const PageObjects = getPageObjects(['common', 'console', 'header']);
+  const testSubjects = getService('testSubjects');
+
+  describe('Painless lab', function describeIndexTests() {
+    before(async () => {
+      await PageObjects.common.navigateToApp('dev_tools', { hash: '/painless_lab' });
+      await retry.waitFor('Wait for editor to be visible', async () => {
+        return testSubjects.isDisplayed('painless_lab');
+      });
+    });
+
+    it('should show the show API request button and click it', async () => {
+      await testSubjects.click('btnViewRequest');
+    });
+
+    it('flyout should appear in page', async () => {
+      const flyoutHeader = await testSubjects.find('flyoutHeader');
+
+      expect(await flyoutHeader.isDisplayed()).to.be(true);
```

Comment:
We have a helper method that allows us to combine these two lines into one:
```
await testSubjects.existOrFail('flyoutHeader', { timeout: 10 * 1000 });
```
The default exist timeout is relatively short (2.5 seconds), so if some loading is involved it's often good to increase the timeout to e.g. 10 seconds (or whatever is sensible for the given step).

Reply:
Pushed both changes.